About.html

```diff
@@ -0,0 +1,118 @@
+                <li><button>CC Campaign</button></li>
+            </ul>
+       
+        </nav>
+</header>
+<div>
+    <div class="headding">
+        <div>.</div>
+      <ul class='Navmenu'>
+        <li class="nav-items"><a class="nav-links" href="./index.html">Home</a> </li>
+        <li class="nav-items"><a class="nav-links" href="./About.html">About</a> </li>
+        <li  class="nav-items"><a class="nav-links"  href="#about">Program</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Join</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Sponser</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">New</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">CC Campaign</a></li>
+      </ul>
+      <div class="hamburger-menu">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+
+    </div>
+</div>
+<section class="intro-sections">
+        <div class="intro-wrapper">
+            <h3 class="intro-wrappers-h3">"The Buganda Pride"</h3>
+            <h1>THE BUGANDA CULTURAL PROSPEROUS <br/>FUTURE FLEX  2022</h1>
+        </div>
+        <p>
+            Buganda is a Bantu kingdom within Uganda. The kingdom of the Baganda people, Buganda is the largest of the traditional kingdoms in present-day East Africa, consisting of Buganda's Central Region, including the Ugandan capital Kampala. The 14 million Baganda (singular Muganda; often referred to simply by the root word and adjective, Ganda) make up the largest Ugandan region, representing approximately 26.6% of Uganda's population.
+        </p>
+        <br>
+        <br>
+        <span>Please contact us via Email for any further questions about the Science Conference 2023</span>
+        <br>
+        <br>
+        <span>culture@bugand-kingdom.org</span>
+</section>
+<section class="intro-sectioned">
+    <h3 class="intro-sectioned-h3">The Buganda KingsMen cultural Galla</h3>
+    <p>
+    The logo of the Global submit was decided through the logo competition form 8th June to 7th July. After online votiong and election commitee's judging process, the logo from Nanjal Gumbson won the price.
+     After consulting Nanjal Gumbson, we optimized the prize winning logo for this years as seen below.
+    </p>
+    <h1 class="kingsman"><span class="uganda">BUGANDA</span> KINGSMEN</h1>
+</section>
+<section class="past">
+<h3>PAST CULTURAL GALLAS</h3>
+<span>Take a look at our past gallas</span>
+<ul>
+    <li>
+        2022 BUGANDA TECH INTEGRATIONS
+    </li>
+    <li>
+        2021 BUGANDA TECH INTEGRATIONS
+    </li>
+</ul>
+
+</section>
+<section class="partners">
+    <h3 class="partners-h3">Partners</h3>
+    <div class="sponsered">
+        <h4 class="gods">KingsMen</h4>
+        <h4 class="kings">Buganda God</h4>
+        <h4 class='gods'>GULU</h4>
+        <h4 class="kings">Buganda Kingdom</h4>
```

Comment:
I kindly suggest in this section to use UL/LI listings so that your site follows good HTML practices.